src/templates/expressions/fields.ts

```diff
@@ -47,6 +48,24 @@ export class StringListField implements TsExpression {
 "${this.fieldName}" : {
     type: "${this.fieldType}";
     value: string[];
+    disabled?: boolean;
+    error?: string;
+};`.trim();
+    }
+}
+
+export class StringFieldInSavedRecord
+    implements TsExpression {
+    constructor(
+        private fieldName: string,
+        private fieldType: string
+    ) {}
+
+    tsExpression(): string {
+        return `
+${this.fieldName} : {
```

Comment:
```suggestion
"${this.fieldName}" : {
```
This line should use double quotation. #86

Reply:
@toshi-toma I fixed it 🙏

Reply:
Thanks!!